This patch suxxs, I'll submit the good one (BSD > 199....) to
libwww maintainer.

BTW, O_SYNC is part of Unix98 compliance and we should have it.

--- libwww/Library/src/HTMemLog.c.orig	Mon Jan 18 00:06:34 1999
+++ libwww/Library/src/HTMemLog.c	Mon Jan 18 00:09:21 1999
@@ -15,7 +15,7 @@
 #include "HTMemLog.h"
 #include "HTTimer.h"
 
-#if defined(WWW_MSWINDOWS) || defined(__FreeBSD__)
+#if defined(WWW_MSWINDOWS) || defined(__FreeBSD__) || defined(__OpenBSD__)
 #define OPEN_FLAGS	O_WRONLY|O_CREAT|O_TRUNC
 #else /* WWW_MSWINDOWS */
 #define OPEN_FLAGS	O_WRONLY|O_CREAT|O_TRUNC|O_SYNC
